phpstan.neon.dist

```diff
@@ -1,4 +1,5 @@
 parameters:
+    checkMissingIterableValueType: false
```

Comment:
Issues related to this setting appeared because phpcbf removed some phpdoc comments